code/modules/mob/living/carbon/carbon.dm

```diff
@@ -101,14 +101,18 @@
 		return FALSE
 
 	if(stun)
```

Comment:
would you change the name of the parameter (also set the bools to TRUE/FALSE while you are at it)